common/src/main/java/tachyon/Constants.java

```diff
@@ -226,6 +226,20 @@
    */
   public static final String WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT =
       "tachyon.worker.tieredstore.level%d.dirs.quota";
+  /**
+   * This format is used as a template to generate the property name for a given level.
+   * e.g., {@code String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_RESERVED_RATIO_FORMAT, level)}
+   */
+  public static final String WORKER_TIERED_STORAGE_LEVEL_RESERVED_RATIO_FORMAT =
+      "tachyon.worker.tieredstore.level%d.reserved.ratio";
```

Comment:
I think its better to have a default reserved ratio in tachyon-default.properties